const.py

```diff
@@ -19,23 +19,27 @@
 be_port = None
 be_location = "/server/"
 
-#config logging
+# config logging
 logging.basicConfig(filename='logs.log', encoding='utf-8', level=logging.INFO)
 
 # command that imports items into oai
 # in github action, this command is correct
-import_command = "docker exec -it dspace /dspace/bin/dspace oai import -c > /dev/null 2> /dev/null"
+import_command = "docker exec -it dspace /dspace/bin/dspace oai import -c > " \
+                 "/dev/null 2> /dev/null"
 # if run on dev-5 under other than devops users (with sudo rights, obviously)
-# import_command = "sudo docker exec -it dspace /dspace/bin/dspace oai import -c > /dev/null 2> /dev/null"
+# import_command = "sudo docker exec -it dspace /dspace/bin/dspace oai import -c >
+# /dev/null 2> /dev/null"
 
 # when run locally on windows. Might need replacement of path
 # import_command = "cd C:/dspace/bin && dspace oai import -c > NUL 2> NUL"
 
 """
 test to find out trivial mistakes
 """
-if use_ssl != expected.exp_SSL or host != expected.exp_host or fe_port != expected.exp_FE_port \
-        or be_port != expected.exp_BE_port or import_command != expected.exp_import_command:
+if use_ssl != expected.exp_SSL or host != expected.exp_host or \
+        fe_port != expected.exp_FE_port \
+        or be_port != expected.exp_BE_port or \
+        import_command != expected.exp_import_command:
     main = "Host settings are not what is expected for tests!!"
```

Comment:
this line should be considered and maybe removed. It was added when this only served as tests. Now it is not necessary relevant

Reply:
it's your code,  I prefer if you remove it.

Reply:
agreed